src/directives/ctr-input.ts

```diff
@@ -99,7 +100,9 @@ export class CtrInput {
             if (this.completer.hasHighlighted()) {
                 event.preventDefault();
             }
-            this.handleSelection();
+            if (!this.forceSelection || this.completer._hasHighlighted) {
```

Comment:
the check can be done inside `handleSelection` which will prevent the code duplication here and in `KEY_TAB`